creations/src/App.js

```diff
@@ -15,12 +15,12 @@ function App() {
                 <Slider />
                 <nav className="navigation">
                     <ul className="list-non-bullet list">
-                        <li className="list-inline"><a className="toplink" href="http:#"> SERVICE </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> CLIENT </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> REVIEW </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> GALLERY </a></li>
-                        <li className="list-inline"><a className="toplink" href="http:#"> ABOUT US </a></li>
-                        <li className="list-inline"><a className="toplink" href="#contact"> CONTACT </a></li>
+                        <li className="list-inline"><a className="toplink" href="#Service-part"> SERVICES </a></li>
```

Comment:
@UmangLadha create custom functional component to accommodate these redundant changes